service/collector/cluster_info.go

```diff
@@ -35,7 +35,7 @@ func (c *Collector) collectClusterInfo(ch chan<- prometheus.Metric, clients []aw
 	for {
 		opts.Continue = continueToken
 
-		clustersConfigList, err := c.g8sClient.ProviderV1alpha1().AWSConfigs("").List(opts)
+		clustersConfigList, err := c.helper.g8sClient.ProviderV1alpha1().AWSConfigs("").List(opts)
```

Comment:
There is some AWS magic that I wrapped into a `Helper` type. This may be rather intermediate since I am not really certain and not really happy with it but that kept me going here now with my gradual transition.